### PR TITLE
Update aware lib

### PR DESCRIPTION
**Pull request by akuvisur**
Updated to latest AWARE lib
Added readme

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,15 +1,29 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
+project.ext {
+    support_libs = (System.getenv("support_libs") as String ?: "23.3.0")
+    google_libs = (System.getenv("google_libs") as String ?: "8.4.0")
+    version_code = 400 + (System.getenv("BUILD_NUMBER") as Integer ?: 0)
+    version_readable = "3.2." + version_code
```
Comment (denzilferreira) [inline build.gradle]: make this 4.0. + version_code

File: device_usage/build.gradle [hunk 2]
```diff
@@ -25,10 +73,11 @@ repositories {
 }
 
 dependencies {
+    compile fileTree(dir: 'libs', include: ['*.jar'])
     compile 'com.android.support:appcompat-v7:23.1.1'
```
Comment (denzilferreira) [inline device_usage/build.gradle]: remove this line, it's repeated below.

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.2.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.2-all.zip
```
Comment (denzilferreira) [inline gradle/wrapper/gradle-wrapper.properties]: The latest is gradle-2.11-all.zip

**Comment by akuvisur** [comment]
Fixed these.

**Pull request merged**